contracts/src/interfaces/IOmniAVSAdmin.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.12;
+
+import { IOmniPortal } from "../interfaces/IOmniPortal.sol";
+
+/**
+ * @title OmniAVSAdmin
+ * @notice Omni AVS admin internface.
+ */
+interface IOmniAVSAdmin {
```

Comment:
was following the pattern I did for omni portal, tho it's less useful here probably, because we are the only ones who need an interface to this  contract rn